olm-catalog/serverless-operator/project.yaml

```diff
@@ -16,6 +16,7 @@ requirements:
   kube:
     minVersion: 1.15.0
   golang: '1.14'
+  nodejs: 14.x
```

Comment:
want to add a comment to explain why Nodejs is needed?

Reply:
PR description isn't enough?